system/jlib/jsocket.cpp

```diff
@@ -1192,6 +1192,7 @@ ISocket* CSocket::accept(bool allowcancel, SocketEndpoint *peerEp)
         getSockAddrEndpoint(peerSockAddr, peerSockAddrLen, *peerEp);
 
     CSocket *ret = new CSocket(newsock,sm_tcp,true);
+    ret->checkCfgKeepAlive();
```

Comment:
perhaps better in ctor? to cover other existing paths and any future ones that construct like this.
( e.g. the one in ISocket::attach )

Reply:
We decided ok for now to leave in accept().